tools/ticket_generator/node/wallet_test.go

```diff
@@ -0,0 +1,99 @@
+package node
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pastelnetwork/gonode/tools/ticket_generator/config"
+	"github.com/pastelnetwork/gonode/walletnode/services/artworkregister"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUploadImage(t *testing.T) {
```

Comment:
we've standardized tests to use table-driven approach. Pls follow

https://dave.cheney.net/2019/05/07/prefer-table-driven-tests
https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go